.circleci/workflows.yml

```diff
@@ -2522,10 +2565,13 @@ jobs:
 linux-x64-workflow: &linux-x64-workflow
   jobs:
     - node_modules_install
+    - build-better-sqlite3:
+        requires:
+          - node_modules_install
```

Comment:
My only concern here is that we're adding like a 4-minute blocking job to the top of our already extremely long workflow. How often is this dependency going to change? Could we at least cache it in CircleCI? Seems unnecessary to re-build it every single time

Reply:
@astone123 good call. I reswizzled a bit and now things are caching. Basically the node modules cache now depends on the centos image as well. And we only build better sqlite if the version of electron or better sqlite3 has changed.